src/lib/event/EventFactory.ts

```diff
@@ -170,6 +191,7 @@ class EventFactory implements IEventFactory {
       page_url: globalThis.location.href,
       library_name: "Formo Web SDK",
       library_version,
+      is_brave_browser: isBraveBrowser,
```

Comment:
make this more generic:

browser: 'brave'

in case we need to support detection for other browsers